crates/rattler_installs_packages/src/wheel.rs

```diff
@@ -819,7 +820,7 @@ mod test {
         let vitals = wheel.get_vitals().unwrap();
 
         // Construct the path lookup to install packages to
-        let install_paths = InstallPaths::for_venv((3, 8), false);
+        let install_paths = InstallPaths::for_venv(PythonInterpreterVersion::new(3, 8, 5), false);
```

Comment:
Maybe it would be nice to implement `From<(u32, u32, u32)>` so you can just do `(3,8,5).into()`? WDYT?

Reply:
Sure was thinking about it as well, will add it.

Reply:
Done!